.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Welcome to the contribution guide 🤗
+
+We are excited to invite the community to contribute to the repository! We appreciate all contributions, big or small. Your efforts help make this repository a valuable resource for everyone working with Llama 3.x models in the Hugging Face ecosystem.
+
+Thank you for your time and happy coding!
+
+## 🚀 How to Contribute
+
+1. **Fork the Repository**
+    - Click on the "Fork" button at the top right corner of this page to create your own copy of the repository.
+    
+    ![fork button](../assets/Fork.png)
+
+2. **Create a Branch**
+    - In your forked repository, create a new branch for your contribution:
+        ```bash
+        git checkout -b feature/your-feature-name
+        ```
+
+3. **Make Your Changes**
+    - Add your scripts, notebooks, or any relevant files.
+    - **Don't forget to update the `README.md`** to include your example, 
+        so others can easily find and use it.
+
+4. **Commit and Push**
+    - Commit your changes with a meaningful commit message:
+        ```bash
+        git commit -m "Add feature: your feature name"
+        ```
+    - Push the changes to your forked repository:
+        ```bash
+        git push origin feature/your-feature-name
+        ```
+
+5. **Open a Pull Request**
+    - Navigate to the original repository and click on "New Pull Request"
+    - Compare across forks and select your branch.
+    
+    ![pull request](../assets/PR.png)
+    - Provide a clear description of your contribution.
+   
+
+## 💡 Need Help?
+
+If you have any questions or need guidance, feel free to open an issue or start a discussion. We're here to help!
```

Comment:
Let's concentrate discussions on issues/PRs. I think we should not enable discussions for this repo as it's too small at the moment.

```suggestion
If you have any questions or need guidance, feel free to open an issue or draft PR. We're here to help!
```